packages/google_maps_flutter/google_maps_flutter/README.md

```diff
@@ -98,7 +101,16 @@ import GoogleMaps
 }
 ```
 
-### Both
+### Web
+
+You'll need to modify the `web/index.html` file of your Flutter Web application
+to include the Google Maps JS SDK.
+
+Check [package:google_maps_flutter_web README](https://pub.dev/packages/google_maps_flutter_web)
```

Comment:
How about "The `google_maps_flutter_web` README" as the link text.

Reply:
Yes, I like it better, also added the backticks around the package name to make it clearer.